utils/stats.py

```diff
@@ -147,8 +154,30 @@ def stats_card(
         # move the cursor to the beginning
         image_binary.seek(0)
 
-        file = discord.File(fp=image_binary, filename=f"{leetcode_id}.png")
+        file = discord.File(fp=image_binary, filename=f"{filename}.png")
 
     os.remove(paths[0])
 
     return file
+
+def anonymise_stats_card(path: str) -> None:
```

Comment:
As pillow is not async, and therefore blocks the thread, we need to execute it in a separate thread. To do that, use the to_thread decorator